backend/models/bookstore.js

```diff
@@ -12,7 +12,16 @@ const BookstoreSchema = new mongoose.Schema({
   coffee: {type: Boolean, default: false},
   wifi: {type: Boolean, default: false},
   plugs: {type: Boolean, default: false},
-  events: {type: Boolean, default: false}
+  events: {type: Boolean, default: false},
+  twitter: String,
+  facebook: String,
+  instagram: String,
+  addressCountry: String,
```

Comment:
should be an object called address, with fields inside of it, no?

Reply:
Look for an express address schema module on npm. There ought to be one we can use.

Reply:
I did not not find a schema module with a global reach. I updated the schema per ![schema.org](http://schema.org/PostalAddress).